stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessageContent.kt

```diff
@@ -20,32 +20,50 @@ import androidx.compose.foundation.layout.Row
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.getstream.sdk.chat.utils.extensions.isMine
+import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.models.Message
 import io.getstream.chat.android.compose.ui.attachments.content.QuotedMessageAttachmentContent
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
 
 /**
  * Represents the default quoted message content that shows an attachment preview, if available, and the message text.
  *
- * @param message The message to show.
+ * @param message The quoted message to show.
  * @param modifier Modifier for styling.
+ * @param replyMessage The message that contains the reply.
  * @param attachmentContent The content for the attachment preview, if available.
  * @param textContent The content for the text preview, or the attachment name or type.
  */
 @Composable
 public fun QuotedMessageContent(
     message: Message,
```

Comment:
Better to rename `message` to `quotedMessage`, but since the PR is targeting v5, don't want to deprecate it.